packages/components/scss/BaseStyleSheet.scss

```diff
@@ -9,12 +9,7 @@
   color: var(--dh-color-text);
 
   .spectrum-theme-provider {
-    // This is important for portals with rounded corners (e.g. Popover) so that
-    // the underlying background color shows.
-    --dh-spectrum-theme-provider-bg: unset;
-
-    background-color: var(--dh-spectrum-theme-provider-bg);
-    color: var(--dh-color-text);
+    display: contents;
```

Comment:
seems like this would probably break something else, as that existing code was serving a purpose.

Reply:
I've reverted the change as it wasn't needed after wrapping the `Modal` in `SpectrumThemeProvider`.